nextapp/lib/DataManager.ts

```
@@ -0,0 +1,2085 @@
+                throw new Error(`Vehicle ${vehicleId} needs at least 3 images for reel generation`);
+              }
+              break;
+            
+            case 'marketing':
+              processingResult = await this.generateMarketingContent(vehicleId, imageUrls, options);
+              break;
+            
+            case 'all':
+              // Process all types sequentially
+              const allResults = {};
+              
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)